src/ClimaAtmos/Parameterizations/TurbulenceConvection/test/runtests.jl

```diff
@@ -0,0 +1,9 @@
+using Test
+
+for submodule in ["Grids",
+                  "StateVecs",
+                  ]
+
+  println("Testing $submodule")
+  include(joinpath("../src",submodule,"runtests.jl"))
```

Comment:
I think we should keep all tests under `test` directories (i.e. move `src/ClimaAtmos/Parameterizations/TurbulenceConvection/src/Grids/runtests.jl` to `src/ClimaAtmos/Parameterizations/TurbulenceConvection/test/Grids.jl`)